packages/shared/getComponentName.js

```diff
@@ -34,10 +34,10 @@ function getWrappedName(
   innerType: any,
   wrapperName: string,
 ): string {
-  const functionName = innerType.displayName || innerType.name || '';
+  const innerName = getComponentName(innerType) || '';
```

Comment:
This may be a bit overkill for render functions of `forwardRef`. We could extract the logic of `innerName` to the call-site of `getComponentName`.